bakery/src/tasks/report-state-complete.js

```diff
@@ -24,17 +24,22 @@ const task = (taskArgs) => {
         AWS_SECRET_ACCESS_KEY: `${awsSecretAccessKey}`
       },
       inputs: [
-        { name: 'book' },
-        { name: 'jsonified-book' }
+        { name: 'book' }
       ],
       run: {
         path: '/bin/bash',
         args: [
           '-cxe',
           dedent`
-          collection_id="$(cat book/collection_id)"
-          book_legacy_version="$(cat book/version)"
-          complete_filename=".${statePrefix}.$collection_id@$book_legacy_version.complete"
+          from_archive="$(cat book/server)"
```

Comment:
consider passing in archive vs. git to the task itself, similar to [here](https://github.com/openstax/output-producer-service/blob/0fdd4bd61acb4231f263ee0a61bc44cd50616e02/bakery/src/tasks/look-up-book.js#L26), so pipeline behavior is always consistent from job to job.